docs/schema-generator/customizing-schemas/advanced-features.md

```diff
@@ -8,25 +8,24 @@ title: Advanced Features
 There are a couple ways you can add more types to the schema without having them be directly consumed by a type in your schema.
 This may be required for [Apollo Federation](federated/apollo-federation), or maybe adding other interface implementations that are not picked up.
 
+### `SchemaGenerator::generateSchema`
```

Comment:
I accidentally pushed this doc fix in this PR as well -> @smyrick let me know if you want me to move this to separate commit

Reply:
If this PR is all doc ena example updates, it's fine to merge as one change.